client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -849,7 +842,8 @@
     }
 
     var cleanHtml = function(data) {
-        return data.replace(/<\/?[^>]+>/gi, '').replace('&nbsp;', ' ');
+        return data.replace(/<br>/g, '&nbsp;').replace(/<\/?[^>]+><\/?[^>]+>/gi, ' ')
```

Comment:
What about e.g. `<br/>` or `<br />`(note the space)? Could these also appear in data?